CMakeConfig.txt

```diff
@@ -56,6 +56,7 @@ set(PROTOCOL_LM76 ON CACHE BOOL "support for the lm76 i2c temperature sensor")
 set(PROTOCOL_LIRC ON CACHE BOOL "support for the Lirc API")
 set(PROTOCOL_LIVOLO_SWITCH ON CACHE BOOL "support for the Livolo switch protocol")
 set(PROTOCOL_LOGILINK_SWITCH ON CACHE BOOL "support for the Logilink switch protocol")
+set(PROTOCOL_LOGILINK_SWITCH2 ON CACHE BOOL "support for the Logilink EC0003 switch protocol")
```

Comment:
Can you check what type was supported by the other logilink protocol?

Reply:
Logilink EC0002, EC0004, EC0005 and EC0006

Reply:
Maybe we should add these version numbers to the other protocol as well:
`support for the Logilink switch protocol EC0002,4,5,6`